Cargo.toml

```diff
@@ -5,22 +5,27 @@ authors = ["Victor Berger <victor.berger@thalesgroup.com>"]
 license = "MIT"
 
 [dependencies]
-wayland-server = "0.9.1"
+wayland-server = "0.9.4"
 nix = "0.7.0"
 xkbcommon = "0.2.1"
 tempfile = "2.1.5"
 slog = { version = "2.0.0" }
 slog-stdlog = "2.0.0-0.2"
-glutin = { version = "~0.7.4", optional = true }
+libloading = "0.4.0"
+wayland-client = { version = "~0.8.6", optional = true }
```

Comment:
Yes when I set both to use dlopen.
And wayland-client 0.8 is sadly required, as winit is still expecting that.